sklearn/feature_selection/_rfe.py

```diff
@@ -324,7 +324,14 @@ def score(self, X, y):
 
         y : array of shape [n_samples]
             The target values.
+
+        Returns
+        -------
+        score : float
+            Score of the underlying base estimator computed with the selected
+            features returned by `rfe.transform(X)` and `y`.
         """
+
```

Comment:
This extra line is not needed and not related to fixing the content of the docstring itself I believe.

```suggestion
```